src/scaraplate/rollup.py

```diff
@@ -14,7 +14,7 @@
 from .template import TemplateMeta, get_template_meta_from_git
 
 
-__all__ = ("rollup",)
+__all__ = ("rollup", "InvalidScaraplateTemplateError")
```

Comment:
This is the only exception we want to public? I think we don't any.

Reply:
This is the only custom exception as for now, so there's nothing else to publish.

Reply:
`__all__` means public. Public means to preserve backward compatibility. There are some holywars about what to count as public on python ML, but I would like to reduce a number of those occurances for us if we don't mean such.

Reply:
> __all__ means public. Public means to preserve backward compatibility. 

Correct. And I indeed want to expose this exception as a part of the public API (and keep it in the future).

The goal is to allow scaraplate to be used in Python, and for that one might want to import this exception to process it differently.

Reply:
Good call. Than it's ok.

Reply:
But in this case exception shouldn't be declarared at the end of file.

Reply:
> But in this case exception shouldn't be declarared at the end of file.

Fair enough. Moved to the beginning of the file and added a simple docstring.